src/Templates/WorkflowTemplate.php

```diff
@@ -70,6 +70,8 @@
  */
 class WorkflowTemplate
 {
+    // These are loosely typed to allow for easy conversion from YAML and backwards compatibility
+    // e.g. version is likely to be an int, though the DB column it goes into is a Varchar
```

Comment:
```suggestion
```
YAML is capable of type inference. I'm fine with not adding strong typing in this update, but I don't think we should include this comment as it implies we _shouldn't_ add strong typing here at any point.

Reply:
I'm not sure why you mean by YAML is capable of type inference

The intention here is that developers are likely to define things with the wrong type in YAML, e.g. in workflows.yml we have `- 0.2` which is used as a $version param in `WorkflowTemplate::__construct()`, ironically the default value of that param is `'0.0'`.  The column it goes into is a varchar.